### PR TITLE
[src] Share the NSTextAttachment[Container] implementations between AppKit and UIKit.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: src/xkit.cs [hunk 1]
```diff
@@ -2090,4 +2093,78 @@ interface NSLayoutConstraint
...
+	[Mac (10,11)]
+	[MacCatalyst (13,0)]
+	[Model]
+	[Protocol]
+	[BaseType (typeof (NSObject))]
+	partial interface NSTextAttachmentContainer {
+		[Abstract]
+		[Export ("imageForBounds:textContainer:characterIndex:")]
+		[return: NullAllowed]
+#if MONOMAC && !XAMCORE_4_0
```
Comment (Therzok) [inline src/xkit.cs:2107]: Is the macOS target not going to work anymore in XAMCORE 4?

Looks like it would generate UIImage

Comment (Therzok) [reply src/xkit.cs:2107]: Or was this supposed to return `Image` as it's alias'd above

Comment (rolfbjarne) [reply src/xkit.cs:2107]: Fixed to use the alias'ed definition above. Thanks!

**Comment by rolfbjarne** [comment]
Test failure is unrelated (https://github.com/xamarin/maccore/issues/2443).

**Review by spouliot — APPROVED** [review]
API diff for macOS shows some modifications for parameter names - but we generally prefer consistence (and sharing) over strict binary compatibility

**Pull request merged**